### PR TITLE
Add flowers to loadout

### PR DESCRIPTION
**Pull request by JustArt1m**
## About the PR
Added poppy and lily head flowers.
Added flower wreath, poppy flower and lily flower to loadout.

## Why / Balance
Just for more customization of  loadout.

## Media
Screenshot 2024-10-06 124244
Screenshot 2024-09-15 232353

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl: Just_Art
- add: Added a flower wreath and hairflower to loadout!

### CODE REVIEW

**Comment by beck-thompson** [comment]
See: #30079 and #29302 for more discussion.

**Comment by JustArt1m** [comment]
> See: #30079 and #29302 for more discussion.

So I will still need to make second versions of these flowers in order to exclude the possibility of using them outside the cosmetic context?

**Comment by beck-thompson** [comment]
> So I will still need to make second versions of these flowers in order to exclude the possibility of using them outside the cosmetic context?

Unknown. The maintainers switched back and fourth between which version they wanted 🤷
I love the idea so I just hope it gets added in some version!

**Comment by slarticodefast** [comment]
> Unknown. The maintainers switched back and fourth between which version they wanted 🤷

That maintainer discussion was way too long last time. Medical flowers might cause metagaming issues as they have a significant amount how healing reagents inside them and loadouts should be mostly for visual custumization. The suggestion we finally came up with was that another "hairflower" item with a new item sprite should be made (in contrast to a "fake poppy"), similar to the one that had been removed when flowers were wearable. The worn sprite could the same as the poppy's.

**Comment by SlamBamActionman** [comment]
What Slart said. In addition, since Lillies are a mutation botanists have to work towards, I personally don't think they should be a roundstart cosmetic item. Makes botany still have a reason to make flowers for cosmetic purposes.

**Comment by Boaz1111** [comment]
> What Slart said. In addition, since Lillies are a mutation botanists have to work towards, I personally don't think they should be a roundstart cosmetic item. Makes botany still have a reason to make flowers for cosmetic purposes.

Agree with Slam on this one, I love making hairflower lilies for the crew, and I think people being able to just spawn with them would be weird.

**Comment by lzk228** [comment]
and we are back to hairflowers... that were removed in #25475

**Comment by slarticodefast** [comment]
> and we are back to hairflowers... that were removed in https://github.com/space-wizards/space-station-14/pull/25475

We did not have loadouts back then

**Comment by bruhmogus** [comment]
why are these entire flowers as opposed to just using the normal preexisting poppy and lily?

**Comment by JustArt1m** [comment]
> why are these entire flowers as opposed to just using the normal preexisting poppy and lily?

https://github.com/space-wizards/space-station-14/pull/32097#issuecomment-2345752439

**Comment by slarticodefast** [comment]
Ok, you might have misunderstood my comment, sorry if I was unclear. The loadout hairflower should not be a poppy at all, since this causes confusion with the botany plant. The new sprite makes this even worse: The hair flower is a whole plant while the botany plant is only a flower (you would not put a whole plant stem into your hair). Also I agree with Slam that the lily should be only available through botany.

So to reiterate, could you:
- remove the lily
- Resprite the item icon for the hairflower, so that it does not resemble the poppy, but some other, unspecific red flower
- rename it to "hairflower"

The worn sprite for the hairflower and the flower wreath can stay the way they are.

**Comment by JustArt1m** [comment]
@slarticodefast

**Review by UbaserB — COMMENTED** [review]
can you make the middle of the flower sprite a bunch darker please

File: Resources/Prototypes/Entities/Clothing/Head/misc.yml [hunk 1]
```diff
@@ -312,3 +312,14 @@
       - WhitelistChameleon
   - type: StaticPrice
     price: 1
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatHairflower
+  name: hairflower
+  description: A beautiful hairflower that can be inserted between the locks of hair.
```
Comment (UbaserB) [inline Resources/Prototypes/Entities/Clothing/Head/misc.yml:320]: 'between locks of hair' sounds better, just scratch the 'the'

**Comment by JustArt1m** [comment]
@UbaserB ?

**Comment by JustArt1m** [comment]
@UbaserB What is the result?

**Comment by UbaserB** [comment]
> @UbaserB What is the result?

Sorry for the very late reply, can you make the flower a bit smaller?

**Comment by JustArt1m** [comment]
> > @UbaserB What is the result?
> 
> Sorry for the very late reply, can you make the flower a bit smaller?

Sprite or item?

**Comment by UbaserB** [comment]
> Sprite or item?

item

**Comment by UbaserB** [comment]
Do not ever mess around with the scale, always edit the sprite itself.

**Review by UbaserB — APPROVED** [review]
Good enough for now.

**Pull request merged**